frontend/src/components/toolbar/problem-metadata/problem-metadata.component.tsx

```diff
@@ -110,6 +135,21 @@ export function ProblemMetadataComponent() {
       >
         Poprawić
       </Button>
+      {alertSnackbar(
+        "Wczytany harmonogram nie zawiera ilości pielęgniarek lub opiekunek",
```

Comment:
Ja bym przepisał to mniej więcej w taki sposób, ponieważ teraz jest za dużo kodu który się powtarza:

```
[message, setMessage] = useState()
[opened, setOpened] = useState()



function showToast(message) {
       setMessage(message)
       setOpened(true)
}

function hideToast() {
   setOpened(false)
}


.....

<SnackbarComponent alertMessage={message} open={open} setOpen={setOpen} key={message} />;
```

I potem w opdowiednich miejscach:
```
....
  if (sitterCount < sitterInScheduleCount) {
     showToast("Wprowadzono mniej opiekunek niż w harmonogramie")
....
```

Reply:
Pewnie o tym pytałeś, a ja źle zrozumiałem